index.html

```diff
@@ -957,6 +959,191 @@ <h4>Error Response</h4>
+          a <code>values</code> member which is empty, contains an invalid <a>Property</a> name, the name of a
+          <a href="https://www.w3.org/TR/wot-thing-description/#td-vocab-readOnly--DataSchema"><code>readOnly</code></a>
+          [[wot-thing-description11]] <a>Property</a>, or a value which does not conform to the data schema of the
+          corresponding PropertyAffordance then it MUST send an <a href="#error-response">error response</a>
+          to the <a>Consumer</a> with <code>status</code> set to <code>400</code>.
+        </p>
+        <pre class="example" title="Example writeallproperties Bad Request error response message">
+          <code>
+            {
+              "thingID": "https://mythingserver.com/things/mylamp1",
+              "messageID": "6dcfce6a-b0cd-48e7-992a-b96d5cb2b872",
+              "messageType": "response",
+              "operation": "writeallproperties",
+              "error": {
+                "status": "400",
+                "type": "https://w3c.github.io/web-thing-protocol/errors#400",
+                "title": "Bad Request"
+                "detail": "No property found with the name 'volume'"
+              }
+              "timestamp": "2025-08-13T18:31:43.531Z",
+              "correlationID": "551aaa52-ff95-4fb5-a657-2b5f430d47ff"
+            }
+          </code>
+        </pre>
+        <p>If a <a>Thing</a> receives a <code>writeallproperties</code> request message from a <a>Consumer</a> with
+          a <code>values</code> member containing valid names and values for all writeable <a>Properties</a>, but the
+          writing of at least one of the properties unexpectedly fails, then it MUST send an
+          <a href="#error-response">error response</a> to the <a>Consumer</a> with <code>status</code> set to
+          <code>500</code>.
+        </p>
+        <pre class="example" title="Example writeallproperties Internal Server Error error response message">
+          <code>
+            {
+              "thingID": "https://mythingserver.com/things/mylamp1",
+              "messageID": "35caf0d3-05f6-4a15-8ddf-a765b5280b06",
+              "messageType": "response",
+              "operation": "writeallproperties",
+              "error": {
+                "status": "500",
+                "type": "https://w3c.github.io/web-thing-protocol/errors#500",
+                "title": "Internal Server Error"
+                "detail": "Writing the 'level' property unexpectedly failed"
+              }
+              "timestamp": "2025-08-13T19:04:36.426Z",
+              "correlationID": "551aaa52-ff95-4fb5-a657-2b5f430d47ff"
+            }
+          </code>
+        </pre>
+        <p>If a <code>writeallproperties</code> operation only partially fails (i.e. some <a>Properties</a> can be
+          written but others can not), a <a>Thing</a> MAY include the successfully written values in the error
```

Comment:
why is this "loose". I would have expected a strong language here.

Reply:
I've changed this to MUST